Makefile

```diff
@@ -49,10 +49,8 @@ ifeq ($(STATIC),1)
 TAGS += netgo
 # The installsuffix makes sure we actually get the netgo build, see
 # https://github.com/golang/go/issues/9369#issuecomment-69864440
-GOFLAGS  += -installsuffix netgo
-LDFLAGS += -extldflags "-lm -lstdc++ -static"
-# rebuild everything.
-GOFLAGS += -a
+GOFLAGS += -installsuffix netgo
+LDFLAGS += -extldflags "-static"
```

Comment:
it would build but the outcome would still be dynamically linked. But I didn't really know what I was doing, so if you've tried this inside of a docker container and the output was a statically linked binary, I'm happy.